### DISCUSSION TITLE
Door Lock

### DISCUSSION THREAD

**Discussion by SleekSycko** [Vdauphin/HeartsAndMinds#1356] [Question and answer]
How does one lower the time for opening a locked door with the wire cutters? I've looked in the door sqf. and not found an answer. would appreciate the help.

**Comment by SleekSycko** [comment]
Nvm. I found the door unlock time in the mission sqf.

**Comment by Vdauphin** [comment]
For people who didn't find it: https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/def/mission.sqf#L676

Also you can use directional mine like claymore to open door
